tests/unit/test_tracing.c

```diff
@@ -1,3 +1,4 @@
+#include "sentry.h"
```

Comment:
as for the test failure, maybe try removing this include here.
`sentry_testsupport.h` should pull that in as well. Unfortunately, the include order matters in C, and also if you have some `#define`s before `#include`s, they also influence what happens, which I think is the case here.

Don’t really have a better explanation, but a hunch that this might help, I think I have seen something like this before.